hardware/Limitless.cpp

```diff
@@ -167,10 +167,6 @@ CLimitLess::CLimitLess(const int ID, const int LedType, const int BridgeType, co
 	Init();
 }
 
-CLimitLess::~CLimitLess(void)
-{
```

Comment:
I added this back, each class should have a deconstructor in .cpp file as it's easier for possible additions
I have no problem with a empty one

Reply:
The goal is size reduction.